provision/bin/type-boot.sh

```diff
@@ -4,14 +4,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+echo "docker rm  to remove stopped container"
+docker rm $(docker ps -a -q) || echo "only rm stopped container, ignore rm failure"
```

Comment:
These scripts are used as examples for provisioning DC/OS clusters in general. So I'm trying to keep them as side-effect-free as possible. Is there a reason to kill these if they're running? Leaving them running should be fine, right?